ui/packages/catalog-ui-search/src/main/webapp/react-component/container/workspace-interactions/workspace-interactions.tsx

```diff
@@ -139,6 +141,30 @@ class WorkspaceInteractions extends React.Component<Props, State> {
       wait: true,
     })
   }
+  deletionPrompt = () => {
+    const workspace = store.getWorkspaceById(this.props.workspace.id)
+    const security = new Security(Restrictions.from(workspace))
+
+    if (security.isShared()) {
+      var self = this
+      this.props.listenTo(
```

Comment:
✏️ Use `.once`

Reply:
Can't use `.once` here as we're inheriting this method from `WorkspaceInteractions`